packages/contentstack-import/test/integration/assets.test.js

```diff
@@ -0,0 +1,100 @@
+const fs = require('fs');
+const path = require('path');
+const uniqBy = require('lodash/uniqBy');
+const { expect, test } = require('@oclif/test');
+const { test: customTest } = require('@contentstack/cli-dev-dependencies');
+const { messageHandler } = require('@contentstack/cli-utilities');
+const LoginCommand = require('@contentstack/cli-auth/lib/commands/auth/login').default;
+const AddTokenCommand = require('@contentstack/cli-auth/lib/commands/auth/tokens/add').default;
+const RegionSetCommand = require('@contentstack/cli-config/lib/commands/config/set/region').default;
+const ExportCommand = require('@contentstack/cli-cm-export/src/commands/cm/stacks/export');
+
+const defaultConfig = require('../../src/config/default');
+const { modules } = require('../../src/config/default');
+const { getStackDetailsByRegion, getAssetAndFolderCount, cleanUp, deleteStack, getEnvData } = require('./utils/helper');
+const { PRINT_LOGS, IMPORT_PATH } = require('./config.json');
```

Comment:
The `IMPORT_PATH` key is missing in the config.json. Also, we need to read `PRINT_LOGS` from environment variables instead of config.json file